src/main/java/generator/BaseScenarioGenerator.java

```diff
@@ -237,6 +237,8 @@ private void addMainVehicle(Model model) throws NoSuchMethodException, IllegalAc
         vehicle.addSpeedX((float) (50 + rand.nextInt(90)));
         vehicle.addSpeedY(0F);
         vehicle.addDistance(0F);
+        vehicle.addAccelerationY(0F);
+        vehicle.addAccelerationX(0F);
```

Comment:
Trzeba powiedzieć grupie z generatora żeby też tak sobie zrobili bo jak tego nie ma to wychodzi początkowe przyspieszenie (0.65, 0.65) dla follow i auto skręca